### PR TITLE
Query : Get Module Source

### PR DESCRIPTION
**Pull request by parv0888**
## Purpose
Added support for query `GetModuleSource`

## Changes
Added support for query `GetModuleSource`

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [x] I accept the above linked CLA.

Depends On #90

### CODE REVIEW

File: concordium-sdk/src/main/java/com/concordium/sdk/responses/modulesource/ModuleSource.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package com.concordium.sdk.responses.modulesource;
+
+import concordium.ConcordiumP2PRpc;
+import lombok.Builder;
+import lombok.Getter;
+
+/**
+ * Parsed response of
```
Comment (EmilLa1) [inline concordium-sdk/src/main/java/com/concordium/sdk/responses/modulesource/ModuleSource.java:8]: This type of documentation is not really useful for the consumer of the SDK.

**Review by EmilLa1 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -373,6 +379,13 @@ val peers = client.getPeerList(true);
 SemVer version = client.getVersion();
 ```
 
+#### getModuleSource
+```java
+val moduleSource = client.getModuleSource(
```
[the remark this answers is not in the document]
Comment (parv0888) [reply README.md:384]: done

File: concordium-sdk/src/main/java/com/concordium/sdk/responses/modulesource/ModuleSource.java (hunk 1 above)
Comment (parv0888) [reply concordium-sdk/src/main/java/com/concordium/sdk/responses/modulesource/ModuleSource.java:8]: done

File: concordium-sdk/src/main/java/com/concordium/sdk/Client.java [hunk 3]
```diff
@@ -313,6 +314,29 @@ public Semver getVersion() {
...
+        val res = server()
+                .getModuleSource(ConcordiumP2PRpc.GetModuleSourceRequest.newBuilder()
+                        .setBlockHash(blockHash.asHex())
+                        .setModuleRef(moduleRef.asHex())
+                        .build());
+        val bytes = res.getValue().toByteArray();
+
+        if(bytes.length == 0) {
+            throw new Exception(
+                    String.format("Module %s not found at block %s", moduleRef.asHex(), blockHash.asHex()));
```
[the remark this answers is not in the document]
Comment (parv0888) [reply concordium-sdk/src/main/java/com/concordium/sdk/Client.java:334]: done

**Review by EmilLa1 — APPROVED** [review]

**Comment by EmilLa1** [comment]
Closes https://github.com/Concordium/concordium-java-sdk/pull/90, https://github.com/Concordium/concordium-java-sdk/pull/89, https://github.com/Concordium/concordium-java-sdk/pull/88

**Pull request merged**